__tests__/lib/utils/feature-flags-edge.test.ts

```
@@ -174,22 +174,31 @@ describe('feature-flags-edge', () => {
-        loadFeatureFlagsEdge: mockLoadFunction,
+        ...originalModule,
+        loadFeatureFlagsEdge: mockLoadFeatureFlagsEdge,
+        isFeatureEnabledEdge: (key: string) => {
+          const flags = mockLoadFeatureFlagsEdge();
+          const DEFAULT_FLAGS = { enableAuthentication: true };
+          return flags[key] !== undefined ? flags[key] : DEFAULT_FLAGS[key];
+        }
       }));
 
       // Re-import with the mocked version
       const { isFeatureEnabledEdge: mockIsFeatureEnabledEdge } = require('../../../src/lib/utils/feature-flags-edge');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)